web/js/modules/layers/actions.js

```diff
@@ -391,3 +393,24 @@ export function changeGranuleSatelliteInstrumentGroup(id, granulePlatform) {
     });
   };
 }
+
+export function updateLayerCollection(value) {
```

Comment:
It is unclear to me what data "value" represents here. Can we provide a function definition? Or perhaps we can just pass the id into this function instead?

Reply:
I've changed this to `id` and extracted the id from the object when we dispatch the action in `layerbuilder.js`.